src/components/z-carousel/index.tsx

```diff
@@ -9,10 +9,24 @@ import { Component, h } from "@stencil/core";
 @Component({
   tag: "z-carousel",
   styleUrl: "styles.css",
-  shadow: false
+  shadow: false,
 })
 export class ZCarousel {
+  /** sets whether the z-carousel is on loading state */
+  @Prop() isloading: boolean;
+  /** sets the height of z-carousel ghost loading */
+  @Prop() ghostloadingheight: string;
```

Comment:
If possible I'd like it if we could document somewhere that this prop is mandatory when `isloading` is set to true, as otherwise the component won't show.